app/sequencescape/sequencescape/api/v2/tag_layout_template.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+# tag layout template resource
+class Sequencescape::Api::V2::TagLayoutTemplate < Sequencescape::Api::V2::Base
+  has_one :tag_group
+  has_one :tag2_group, class_name: 'TagGroup'
+
+  def dual_index?
+    tag2_group.present?
+  end
+
+  # Performs the coercion of this instance so that it behaves appropriately given the direction
+  # and walking algorithm information.
+  def coerce
+    extend("limber/tag_layout_template/in_#{direction.gsub(/\s+/, '_')}s".camelize.constantize)
+    extend("limber/tag_layout_template/walk_#{walking_by.gsub(/\s+/, '_')}".camelize.constantize)
+  rescue NameError => e
+    Rails.logger.warn("Unrecognised layout options: #{e.message}")
+    extend Unsupported
```

Comment:
So this method and the one below are taken from the existing `app > models > limber > tag_layout_template.rb` file... is that file needed any more?

Reply:
I'm leaving removal of V1 until the end.  It's still unclear how tightly tied they are into the system and I do still need it until we remove the TODO items.